### PR TITLE
Minor test adjustments

### PR DESCRIPTION
**Pull request by rtretyak**
- fix server42 logstash issues
- fix grpc tests comparing strings instead of integers when picking versions
- add better cleanup for fl outage test
- comment default rules verification with link to defect

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy [hunk 1]
```diff
@@ -104,6 +104,7 @@ class FloodlightKafkaConnectionSpec extends HealthCheckSpecification {
     def "System can detect switch changes if they happen while Floodlight was disconnected after it reconnects"() {
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy:104]: `@Tidy`?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy [hunk 2]
```diff
@@ -121,7 +123,8 @@ class FloodlightKafkaConnectionSpec extends HealthCheckSpecification {
             assert islUtils.getIslInfo(isls, isl.reversed).get().state == IslChangeType.FAILED
         }
 
-        and: "Cleanup: restore the broken link"
+        cleanup:
+        regionsOut && regions.each { lockKeeper.reviveFloodlight(it) }
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy:127]: how do we know that `floodlight` is available again? probably we should add extra check like `assert northbound.activeSwitches.size() == topology.activeSwitches.size() `

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy (hunk 1 above)
Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy:104]: done

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy (hunk 2 above)
Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/FloodlightKafkaConnectionSpec.groovy:127]: done

**Pull request merged**